### PR TITLE
Update lib

### PR DESCRIPTION
**Pull request by xwz123**

### CODE REVIEW

File: lgtm.go [hunk 1]
```diff
@@ -92,12 +93,14 @@ func (bot *robot) addLGTM(cfg *botConfig, e giteeclient.PRNoteEvent, log *logrus
...
-func (bot *robot) removeLGTM(cfg *botConfig, e giteeclient.PRNoteEvent, log *logrus.Entry) error {
-	pr := e.GetPRInfo()
-	org, repo, number := pr.Org, pr.Repo, pr.Number
+func (bot *robot) removeLGTM(cfg *botConfig, e *sdk.NoteEvent, log *logrus.Entry) error {
+	org, repo := e.GetOrgRepo()
+	number := e.GetPRNumber()
 
-	if commenter := e.GetCommenter(); pr.Author != commenter {
-		v, err := bot.hasPermission(commenter, cfg.CheckPermissionBasedOnSigOwners, pr, cfg, log)
+	if commenter := e.GetCommenter(); e.GetPRAuthor() != commenter {
```
Comment (xwz123) [inline lgtm.go:100]: s/commenter := e.GetCommenter(); e.GetPRAuthor() != commenter/e.GetCommenter() != e.GetPRAuthor()

File: permission.go [hunk 2]
```diff
@@ -2,10 +2,10 @@ package main
 
 import (
 	"encoding/base64"
+	sdk "github.com/opensourceways/go-gitee/gitee"
```
Comment (xwz123) [inline permission.go:5]: fmt import

File: lgtm.go [hunk 3]
```diff
@@ -118,7 +121,7 @@ func (bot *robot) removeLGTM(cfg *botConfig, e giteeclient.PRNoteEvent, log *log
 	}
 
 	// the author of pr can remove all of lgtm[-login name] kind labels
-	if v := getLGTMLabelsOnPR(pr.Labels); len(v) > 0 {
+	if v := getLGTMLabelsOnPR(e.GetIssueLabelSet()); len(v) > 0 {
```
Comment (zengchen1024) [inline lgtm.go:124]: check it

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: permission.go [hunk 4]
```diff
@@ -31,27 +31,28 @@ func (bot *robot) hasPermission(
...
 
 	if needCheckSig {
-		return bot.isOwnerOfSig(commenter, pr, cfg, log)
+		return bot.isOwnerOfSig(org, repo, commenter, pr, cfg, log)
 	}
 
 	return false, nil
 }
 
 func (bot *robot) isRepoOwners(
```
Comment (wanghao75) [inline permission.go:45]: isRepoOwners 这个地方是不是不需要了，判断仓库的OWNERS文件，除了community有，其他的库也没有OWNERS文件

Comment (xwz123) [reply permission.go:45]: The goal of this PR is to upgrade the library version without making functional changes

**Pull request merged**